akka-sample-fsm-java/src/main/java/sample/DiningHakkersTyped.java

```diff
@@ -0,0 +1,40 @@
+package sample;
+
+import akka.NotUsed;
+import akka.actor.typed.ActorRef;
+import akka.actor.typed.ActorSystem;
+import akka.actor.typed.Behavior;
+import akka.actor.typed.javadsl.Behaviors;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class DiningHakkersTyped {
+
+    private static Behavior<NotUsed> mainBehavior() {
```

Comment:
Perhaps do a nested static or non nested `MainBehavior` class with `create()` instead to be consistent?

Reply:
:+1: will make his class it rather than call it MainBehavior